appender.go

```diff
@@ -397,6 +414,9 @@ func (a *Appender) flush(ctx context.Context, bulkIndexer *BulkIndexer) error {
 		failedCount[info]++
 		if a.tracingEnabled() {
 			apm.CaptureError(ctx, errors.New(info.Error.Reason)).Send()
+		} else if a.otelTracingEnabled() {
+			// trace.SpanFromContext(ctx).RecordError(errors.New(info.Error.Reason))
+			span.RecordError(errors.New(info.Error.Reason))
```

Comment:
Is possible instead of using a `span` var in the local scope (initialized on line 311) to always extract it from the context. I don't see benefits in doing so but I've yet to finalize a decision on what to use.

Reply:
Decided to use `span` variable.